src/widgets/Notebook.cpp

```diff
@@ -35,11 +35,28 @@ Notebook::Notebook(QWidget *parent)
     this->addButton_->setHidden(true);
 
     this->menu_.addAction(
-        "Toggle visibility of tabs",
+        "Toggle Visibility of Tabs",
```

Comment:
![image](https://user-images.githubusercontent.com/41973452/159988430-1b6159f5-fa06-4b17-9c13-c4643b043ac5.png)

Reply:
Following the existing style for `NotebookTab` actions.

![image](https://user-images.githubusercontent.com/6964154/159988645-a017a834-94b1-42e5-b4f5-720df5ccc617.png)

Channel actions on the other hand are a different case
![image](https://user-images.githubusercontent.com/6964154/159988911-a7ee37b1-81ba-4e07-b67f-f736646eff22.png)

Reply:
but this is not a PR for visibility of tabs now is it 🙂

Reply:
Fair enough